augur/io/file.py

```diff
@@ -2,9 +2,23 @@
 from contextlib import contextmanager
 from io import IOBase
 from textwrap import dedent
-from xopen import PipedCompressionReader, PipedCompressionWriter, xopen
+
+from xopen import xopen
+
 from augur.errors import AugurError
```

Comment:
Remove extra whitespace additions


```suggestion
from xopen import xopen
from augur.errors import AugurError
```

Reply:
This is on purpose, it's in line with isort - I don't think the extra white spaces here are a problem? They don't make the diff significantly bigger overall - and reduce it in the future once we use isort everywhere?

Reply:
I let you decide and merge - no strong opinion